User setup documentation

noahpop77 opened the pull request:
I wrote some documentation that could help others get up and running with KeyConjurer easier. The main README.md file has some context and information on the project but does not have any reproducible steps laid out. The user documentation would help to get people up and running with the project. There is still more to detail but this could help to make KeyConjurer a more user contributor friendly open-source project! 

If you have any advice or feedback I would love to hear it!

punmechanic commented:
Hi Noah, thanks for your contribution. I'm off work until Jan 6th 2025 and will respond to this when I am back :)

noahpop77 commented:
No worries! Hope to be able to help out where I can with KeyConjurer, It seems like a really interesting tool.

noahpop77 commented:
OK I will make the changes to the README.md rather than the setup.md file and resubmit the PR.

noahpop77 commented:
OK @punmechanic I cleaned up the documentation a bit and added it to the current README. Also thanks for letting me know that the documentation for an end user is in the frontend folder. If theres any changes you think would be helpful let me know 😁
